flexbe_states/src/flexbe_states/subscriber_state.py

```diff
@@ -53,8 +54,11 @@ def on_enter(self, userdata):
             self._sub.remove_last_msg(self._topic)
 
     def _connect(self):
-        msg_type, msg_topic, _ = rostopic.get_topic_class(self._topic)
-        if msg_topic == self._topic:
+        global_topic = self._topic
+        if global_topic[0] != '/':
+            global_topic = rospy.get_namespace() + global_topic
+        msg_type, msg_topic, _ = rostopic.get_topic_class(global_topic)
+        if msg_topic == global_topic:
```

Comment:
This looks fine, but there could be some places where we already have something like this done ourselves. 
Cannot judge ATM. Maybe `grep` on `rospy.get_namespace` in our own code so see if there are any uses